versioned_docs/version-1.1.0/configuration.md

```diff
@@ -0,0 +1,14 @@
+---
+id: configuration
+title: Configuration
+---
+
+## Environment Variables
+
+Honeycomb uses environment variables during build and run time to control the different configurations. Below we list the  environment variables used by the app and indicate whether they are mandatory, optional and needed during build (`npm build`) or run time (using the application executable or `npm run dev`)
```

Comment:
I think we should further explain what "setting environment variables" means at build time and run time more explicitly. So for build time, they go in the `env` folder and for run-time they are set in the system. We can provide minimal example on how to do that in windows, max/unix

One way to provide this explanations could be by having a list of the badges we are shooting and having an explanation of the meaning. Something like,

![](https://img.shields.io/badge/-runtime-purple): This badge is used for run-time environment variables . It needs to be set at the system level. Follow [this](setting-environment-variables-systemwide) for OS specific instructions.
![](https://img.shields.io/badge/-buildtime-blue): This badge is used for build-time environment variables. Set in the local `env` folder

Reply:
There's no official link for setting up environment variables system wide. What should I link to?

Reply:
yeah... ideally I guess we create a markdown file containing the info. or we find a decent link - [this](https://www.twilio.com/blog/2017/01/how-to-set-environment-variables.html) is okay for windows but kind of confusing for Mac... we could try to also find official docs? Apple has [this](https://support.apple.com/guide/terminal/use-environment-variables-apd382cc5fa-4f58-4449-b20a-41c53c006f8f/mac) but it only shows how to export for one session... 

or we could just say `It needs to be set at the system level. Use google  for specific OS instructions ;)` @mcmcgrath13 what do you think?

Reply:
what about this link https://www.twilio.com/blog/2017/01/how-to-set-environment-variables.html

Reply:
That's the same I saw. I think it's fine for windows... for Mac they don't say to add to `~/.bash_profile` ... which maybe not the right way always, but naive users won't know when exporting only works for a single session...

Reply:
maybe this one is better https://www.imatest.com/docs/editing-system-environment-variables/#Windows

Reply:
Yeah I think  https://www.imatest.com/docs/editing-system-environment-variables/#Windows looks good